### ISSUE TITLE
Support for Spotify Genres / Moods / Liked songs / ... in library

### ISSUE THREAD

**Issue by timvdsm** [AlexanderBabel/owntone#8]
Hi, many thanks for your work. Owntone is working really great. 
A question or feature request I have:
Would it be possible to support the spotify "Genres & Moods" aswell as the "Liked songs" playlist?

Now under the spotify tab I only see: New releases & Featured Playlist. 

Or is this something I can configure somewhere? 

Many thanks!

**Comment by AlexanderBabel** [comment]
Hey there,

this Docker image should be used as a Spotify Connect solution. For that, I added librespot-java to the image and made it work out of the box with OwnTone. 

You can just open the Spotify app in the same network and select the `Home Assistant` speaker. (Please keep in mind to run the docker container on the host network).

As an alternative, I offer an add-on for Home Assistant OS: https://community.home-assistant.io/t/homepod-connect-spotify-on-homepods-with-spotify-connect/482227

In other words, your specific request is out of scope of the project and should be requested in the upstream OwnTone project: https://github.com/owntone/owntone-server

**Comment by timvdsm** [comment]
Many thanks for clarification. I’ve opened a request on the owntone github.

I also have Home Assistant, so that could be interesting :-) what is your add-on exactly doing? It will create a new Owtone server as an addon in HA? 
I have your docker running on my Synology. And that one is working great. There is also an official owntone integration in HA.
But your addon will have a better integration?

Many thanks

**Comment by AlexanderBabel** [comment]
This add-on just runs OwnTone and librespot-java inside one docker container on Home Assistant OS. You can also just run the container by yourself on your NAS. This add-on does not provide a new integration. You can just use the normal OwnTone Home Assistant integration and control the OwnTone instance inside this image.

The benefit of the add-on/docker image is that it lets you use Spotify Connect without any extra configuration. Just install it, open the OwnTone UI, select the speakers you want to listen to and start music playback through your Spotify app on the `Home Assistant` speaker.
